src/components/PeoplePage.tsx

```diff
@@ -1,29 +1,49 @@
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { Person } from '../types';
+import { useState, useEffect } from 'react';
+import { getPeople } from '../api';
 
 export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [error, setError] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    getPeople()
+      .then(data => setPeople(data))
+      .catch(() => setError(true))
```

Comment:
Consider logging the error in the `catch` block to provide more information about what went wrong. This can help with debugging and understanding the cause of the error.